paddlenlp/transformers/xlnet/tokenizer.py

```diff
@@ -37,44 +37,54 @@
-            to instantiate a tokenizer.
-        do_lower_case (`bool`, optional):
-            Whether to lowercase the input when tokenizing. Defaults to ``False`` and
-            we do not lowercase the input.
-        remove_space (`bool`, optional):
-            Whether to strip the text when tokenizing. Defaults to ``True`` and
-            we remove excess spaces before and after the string.
-        keep_accents (`bool`, optional):
-            Whether to keep accents when tokenizing.
-            Defaults to ``False`` and we don't keep accents.
-        bos_token (`str`, optional):
-            The beginning of sequence token that was used during pretraining. Defaults to ``"<s>"``.
-        eos_token (`str`, optional):
-            The end of sequence token. Defaults to ``"</s>"``.
-        unk_token (`str`, optional):
-            The unknown token. A token that is not in the vocabulary is set to be unk_token
-            inorder to be converted to an ID. Defaults to ``"<unk>"``.
-        sep_token (`str`, optional):
-            The separator token. Defaults to ``"<sep>"``.
-        pad_token (`str`, optional):
-            The token used for padding. Defaults to ``"<pad>"``.
-        cls_token (`str`, optional):
-            The classifier token which is used when doing sequence classification.
-            It is the last token of the sequence when built with special tokens. Defaults to ``"<cls>"``.
-        mask_token (`str`, optional):
-            The token used for masking values. In the masked language modeling task,
-            this is the token used and which the model will try to predict. Defaults to ``"<mask>"``.
-        additional_special_tokens (`List[str]`, optional):
-            Additional special tokens used by the tokenizer. Defaults to ``["<eop>", "<eod>"]``.
+        vocab_file (str):
+            The vocabulary file (ends with '.spm') required to instantiate
+            a `SentencePiece <https://github.com/google/sentencepiece>`__ tokenizer.
+        do_lower_case (bool, optional):
+            Whether or not to lowercase the input when tokenizing. Defaults to `False` and
+            **does not** lowercase the input.
+        remove_space (bool, optional):
+            Whether or not to strip the text when tokenizing. Defaults to `True` and
+            removing excess spaces before and after the string.
+        keep_accents (bool, optional):
+            Whether or not to keep accents when tokenizing. Defaults to `False` and **does not** keep accents.
+        bos_token (str, optional):
+            A special token representing the beginning of a sequence that was used during pretraining.
+            Defaults to `"<s>"`.
+        eos_token (str, optional):
+            A special token representing the end of a sequence that was used during pretraining.
+            Defaults to `"</s>"`.
+        unk_token (str, optional):
```

Comment:
I thought BOS EOS meant "beginning/end of a sentence" (instead of sequence)
However, I am not sure about the design here